blocks/header/header.css

```diff
@@ -234,15 +229,17 @@ header nav .nav-sections ul > li > ul > li {
     position: relative;
   }
 
+    /* This is the style for the dropdown */
   header nav .nav-sections .default-content-wrapper > ul > li[aria-expanded='true'] > ul {
-    display: block;
+    display: flex;
     position: absolute;
     left: -24px;
-    width: 200px;
+    width: 900px;
```

Comment:
have you tested what happens if we have several items inside a category?
<img width="1205" alt="Screenshot 2025-02-10 at 12 33 39" src="https://github.com/user-attachments/assets/5e9804b1-ab68-4bcf-b946-4cf46f6e0399" />

Reply:
@loginesta how are you seeing all these items? I cannot see them in https://lynx-722--aem-boilerplate-commerce--hlxsites.aem.live/

Reply:
I have manually added them editing the DOM in order to see the behaviour of the Site Navigation in the browser